overrides/app/components/algolia/homepage/segmentTracker.js

```diff
@@ -0,0 +1,6 @@
+function trackEvent(event) {
+    if (window && window.analytics) {
```

Comment:
Where is it instantiated?

Reply:
Hello, 
it is initialized here:
https://github.com/algolia/algolia-pwa-demo/blob/57b80470fea1682e1629be3df596da164cfc3a30/overrides/app/components/_app-config/index.jsx